docs/data-create.rst

```diff
@@ -1,73 +1,80 @@
 
-      -- Two-socket server with 2 x 16 Intel Xeon cores, 256 GB memory, and NVIDIA Telsa P100 GPU.
+1. Create folders
+====================================================================
 
-      -- Slide Size: 66816 x 75520 pixels, Magnification: 40x
+Navigate to the base dataset folder
 
-      -- superpixelSize: 64, patchSize: 128
-
-      -- Superpixel segmentation: 40 minutes (only uses CPU)
-
-      -- Feature extraction: 1 hour and 30 minutes (GPU) more than 6 hours (CPU)
+.. code-block:: bash
 
+  $ cd myproject
 
-1. Create your local directories.
+Create subdirectories inside the base folder
 
 .. code-block:: bash
 
-  # Directories below should be ready before creating a new dataset.
-  # svs - a slide image directory name. Users have to place their images to this directory.
-  # A .svs file in "/svs" directory of the docker image is provided as an example.
-  # feature - a feature directory name. Feature data for each slide will be storied in this directory.
-  # boundary - a boundary directory name. Boundary data for each slide will be storied in this directory.
-  # centroid - a centroid directory name. Centroid data for each slide will be storied in this directory.
-  # dataset - a dataset directory name. A dataset will be created in this directory.
-  # create directories for feature, boundary, centroid, and dataset.
   $ mkdir "$PWD"/dataset "$PWD"/feature "$PWD"/boundary "$PWD"/centroid
 
-2. Pull the HistomicsML-TA docker image.
+  Superpixel boundaries and centroids (x,y) coordinates are stored in the *boundary* and *centroid* folders. Features extracted from superpixels are stored in the *feature* folder. The *dataset* folder contains the final transformed data in .h5 format that is ready for ingestion (see below). Finally, an *svs* directory contains whole-slide image files. Data from a single slide is included in the Docker image as an example.
+
+2. Download the dataset creation Docker container
+====================================================================
+
+Download the HistomicsML dataset creation Docker container
 
 .. code-block:: bash
 
   $ docker pull cancerdatascience/hml_dataset_gpu:1.0
 
-3. Create boundaries and centroids of superpixels.
+Use ``SuperpixelSegmentation.py`` to generate superpixel boundaries and centroids
 
 .. code-block:: bash
 
   $ docker run -it --rm --name createboundary -v "$PWD"/boundary:/boundary -v "$PWD"/centroid:/centroid cancerdatascience/hml_dataset_gpu:1.0 python scripts/SuperpixelSegmentation.py --superpixelSize 64 --patchSize 128
 
-.. note:: Adjustable parameters of ``SuperpixelSegmentation.py`` and ``FeatureExtraction.py`` are following::
+.. note:: 
+  Parameters of the superpixel segmentation script ``SuperpixelSegmentation.py`` can be adjusted to change the size, shape, and thresholding of superpixels to discard background regions
```

Comment:
@slee172 I tried to reformat the to improve the rendering. They look OK on the live Sphinx preview (http://livesphinx.herokuapp.com/).

We should include at least the compactedness parameter here to allow people to control this in the segmentation.

Reply:
Added compactedness as a parameter of superpixel segmentation.